docs/core_docs/docs/integrations/text_embedding/voyageai.mdx

```diff
@@ -1,11 +1,19 @@
-# Voyage AI
+## Voyage AI Integration
 
 The `VoyageEmbeddings` class uses the Voyage AI REST API to generate embeddings for a given text.
 
+The `inputType` parameter allows you to specify the type of input text for better embedding results. You can set it to `query`, `document`, or leave it undefined (which is equivalent to `None`).
+
+- `query`: Use this for search or retrieval queries. Voyage AI will prepend a prompt to optimize the embeddings for query use cases.
+- `document`: Use this for documents or content that you want to be retrievable. Voyage AI will prepend a prompt to optimize the embeddings for document use cases.
+- `None` (default): The input text will be directly encoded without any additional prompt.
+
+
 ```typescript
 import { VoyageEmbeddings } from "langchain/embeddings/voyage";
 
 const embeddings = new VoyageEmbeddings({
   apiKey: "YOUR-API-KEY", // In Node.js defaults to process.env.VOYAGEAI_API_KEY
+  inputType: "None", // Optional: specify input type as 'query', 'document', or omit for None
```

Comment:
`None` is weird here?

Reply:
From what I read here :
https://docs.voyageai.com/docs/embeddings
"When the input_type is set to None, and the input text will be directly encoded by our embedding model. Alternatively, when the inputs are documents or queries, the users can specify input_type to be query or document, respectively. "
So sounds good to me ?